### ISSUE TITLE
'fused' argument of BatchNormalization is not saved in the model

### ISSUE THREAD

**Issue by sandeepjana** [tensorflow/tensorflow#40224]
Batch normalization layer's `fused` argument is not part of the saved model h5/json.   

Tested with TF version: 2.1.0 (CPU), Python 3.7.7

```
from tensorflow.keras import layers
from tensorflow.keras.models import Model

x = layers.Input((32, 32))
m = Model(x, layers.BatchNormalization(fused=False)(x))
print('fused' in m.to_json())
```
Output: `False`.
Expected output is `True`.

I found this issue when trying to get equal predictions in different versions of tensorflow.

I made a model with batch normalization layers. I tested the model in TF 1.12 and TF 2.1.0 (both are CPU versions, if that matters) with the same input. I got almost same predictions but differing in 4 or 5th decimal. Difference may become larger if the model is deeper. Once I manually added `"fused" : false` in the model json, the predictions became exactly same, at least after batch norm layer. 

Fixing this issue can help during sanity check of the model, for the developers like me who work with different TF versions simultaneously.

**Comment by amahendrakar** [comment]
Was able to reproduce the issue with [TF v2.2](https://colab.research.google.com/gist/amahendrakar/097019fc33cd11a9261eb368b0db4377/40224.ipynb) and [TF-nightly](https://colab.research.google.com/gist/amahendrakar/5ff8e0e7695d09abc816b7d2b4b812e1/40224-tf-nightly.ipynb). Please find the attached gist. Thanks!

**Comment by ethanyanjiali** [comment]
any updates on this issue?

**Comment by yhliang2018** [comment]
@sandeepjana For your code snippet, the `fused` arg has been set as `False` in the layer, why is it expected to be `True`?

**Comment by saikumarchalla** [comment]
@sandeepjana  Could you please check the above comment and let us know if the issue persists still.

Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by jbboin** [comment]
Hi, this issue is not solved.

When the input is a 4D tensor, serializing and deserializing a `BatchNormalization` layer with `fused=False` will give a layer with `fused=True`. The serialization seems to be the problem. It doesn't save the state of the `fused` parameter. Adding it manually works, as the following snippet (which builds on the snippet posted by @sandeepjana) shows.
Note that unlike the original snippet (3D input, if we count the batch dimension), I added an extra dimension to the input (4D input).

```
from tensorflow.keras import layers
from tensorflow.keras.models import Model

x = layers.Input((32, 32, 1))
m = Model(x, layers.BatchNormalization(fused=False)(x))
print('fused' in m.to_json())

# Load from JSON
from tensorflow.keras.models import model_from_json
m_loaded = model_from_json(m.to_json())
print(m_loaded.layers[1].fused)

# Correct JSON
import json
json_corrected = json.loads(m.to_json())
json_corrected['config']['layers'][1]['config']['fused'] = False
json_corrected = json.dumps(json_corrected)

# Load from corrected JSON
m_loaded = model_from_json(json_corrected)
print(m_loaded.layers[1].fused)
```

This prints:
```
False
True
False
```
which means that:
- the `fused` parameter isn't found in the serialized model
- after deserialization, the BN layer has the parameter `fused = True`, which is incorrect
- after correction of the JSON and deserialization, the BN layer once again has the parameter `fused = False`, which would be the expected behavior.

This was tested in Tensorflow 2.3.0.
